### DISCUSSION TITLE
AgencyOS for Nextjs?

### DISCUSSION THREAD

**Discussion by atilladeniz** [directus-labs/agency-os#96] [Q&A]
Hi Directus Team,
I love the CMS.. but is it possible to have the AgencyOS for Nextjs? and how we can use the Data Collection to a selfhosted Directus Instance ?

**Comment by Abdallah-Awwad** [comment]
There is an unofficial project called "Nextus" that closely mirrors the functionality you're looking for. You can find more details and the repository here: 
https://github.com/luochuanyuewu/nextus
